FF145 Experimental Features: Idempotency-Key (#41698)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

- `security.integrity_policy.stylesheet.enabled`
  - : Set to `true` to enable.

### Idempotency-Key

The {{httpheader("Idempotency-Key")}} HTTP request header can be used by website client code to make a {{HTTPMethod("POST")}} or {{HTTPMethod("PATCH")}} requests {{glossary("idempotent")}} when used with a corresponding server.

> The specification indicates that the endpoints that require this header, the format of the key, and any error responses are defined by the server.

Reviewer: ```suggestion
The specification indicates that the server should document and advertise which endpoints require this header, the format of the key, and expected error responses.
```
